src/sdk/model/o365/o365-organization.ts

```diff
@@ -330,4 +331,23 @@ export class O365Organization extends Entity {
       return json.map(it => new O365BackupRepository(it));
     });
   }
+
+  /**
+   * Get the O365 organization users download in csv format
+   * @param downloadProgress {function} optional progress callback if user wishes to receive progress events
+   * for their download
+   * @returns {Promise<Blob>} will return a promise with the Blob to download
+   */
+  /* istanbul ignore next: autogenerated */
+  async downloadUsers(downloadProgress?: (e: any) => void): Promise<Blob> {
+    return Iland.getHttp().post(`/o365-organizations/${this.uuid}/users-export`, null, {
+      headers: {
+        'Accept': Http.versionMime('application/octet-stream')
+      },
+      responseType: 'arraybuffer',
+      onDownloadProgress: downloadProgress
+    }).then((response) => {
+      return new Blob([response.data], { type: 'application/csv' });
```

Comment:
type 'text/csv' may be [more](https://stackoverflow.com/questions/18927865/whats-the-difference-between-using-application-csv-vs-text-csv) [appropriate](https://stackoverflow.com/questions/393647/response-content-type-as-csv) here, but 'application/csv' also valid.

Reply:
Fixed, thanks @SuperConfuserUser